Sources/AsyncHTTPClient/AsyncAwait/HTTPClientRequest.swift

```diff
@@ -14,10 +14,14 @@
 
 import NIOCore
 import NIOHTTP1
+import NIOSSL
```

Comment:
CI tells us that TLSConfiguration isn't marked as Sendable in Swift 5.6.

```suggestion
#if swift(<5.7)
@preconcurrency import NIOSSL
#else
import NIOSSL
#endif
```

Reply:
That's not the right fix, it'll warn. The right fix is to put it in an `UnsafeTransferBox` in 5.6. The reason this is this way is that on 5.6 we couldn't mark the closures in the TLSConfiguration object as `@preconcurrency Sendable`.

Reply:
@Lukasa it being `var tlsConfiguration: TLSConfig?` --> `var tlsConfiguration: UnsafeTransfer<TLSConfig>?` for 5.6 only?

Reply:
@Lukasa actually I think `@preconcurrency` will work warning-free. That's because the `@preconcurrency` isn't actually unused. But let's see

Reply:
@Lukasa yes worked, all green.